app/src/main/res/layout/fragment_contest_details.xml

```diff
@@ -130,8 +134,8 @@
         android:visibility="gone" />
 
     <include
-        android:id="@+id/view_general_error"
-        layout="@layout/general_error_layout"
+        android:id="@+id/loading_view"
+        layout="@layout/loading_screen_layout"
```

Comment:
You should add the loading view as a new view and not replace the error view.

Reply:
Ok sorry for the inconvenience I will resolve this

Reply:
No worries! Also could you please update your branch too? so that tests can pass. I have updated some of the code. Thanks.

Reply:
I don't know how to update branch I'm little bit new to open source. Could you please tell me?

Reply:
Yeah I updated the branch

Reply:
sure just run these below commands:
```
git remote add upstream https://github.com/cdhiraj40/LeetDroid.git
git pull upstream development
```

An explanation for the commands
1. The command adds the main repository as a remote called "upstream"(you can name anything), so later you can fetch/update code right from your terminal
2. It pulls the code from the development branch of the remote upstream which is the main repo here.

Hope you understood, let me know if you need more explanation!

Reply:
> sure just run these below commands:
> 
> ```
> git remote add upstream https://github.com/cdhiraj40/LeetDroid.git
> git pull upstream development
> ```
> 
> An explanation for the commands
> 
> 1. The command adds the main repository as a remote called "upstream"(you can name anything), so later you can fetch/update code right from your terminal
> 2. It pulls the code from the development branch of the remote upstream which is the main repo here.
> 
> Hope you understood, let me know if you need more explanation!

Thanks for the explanation. I followed these steps but I don't know whether the branch is updated or not.

Reply:
You already have updated the branch so no need to worry 😄

Reply:
Ok did you undo the required changes or have to do it?

Reply:
no you will need to do it

Reply:
Ok